recipe/add-error-when-not-activated.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/selenium/webdriver/common/selenium_manager.py b/selenium/webdriver/common/selenium_manager.py
```

Comment:
Hm, we _just_ got _out_ of the patch-carrying business, and I don't relish having to go back to keeping them up-to-date vs upstream: once upstream offered _a_ way to set it, pretty much had to accept the activation route over patching (which even then still had to rely on an accurate `PATH`, `CONDA_PREFIX` or _something_ to keep this `noarch: python`)...

Reply:
Ya, of course, whatever works.

I'm slightly confused because my changes don't undo the noarch. Is the idea that you want to stay noarch while turning on Windows/MacOS tests?

Reply:
I just thought of a clever and awful way to emit my warning without resorting to patches...

We could replace the vendored binaries with error-emitting Python scripts. That's precisely the codepath we want to target.